RobotTypes: 
  - robot_type1
  - robot_type2
  - ZLQ

robot_type1:
  robot_num: 1
  velocity: 50 # m/h
  move_power: 0.3
  move_temperature: 10
  cold_down_temperature: -30 # origin -10.0
  cold_down_power: 0.1
  max_temperature: 55
  task_file: TaskGraph1

robot_type2:
  robot_num: 1
  velocity: 20 # m/h
  move_power: 0.3
  move_temperature: 10
  cold_down_temperature: -30 # origin -10.0
  cold_down_power: 0.1
  max_temperature: 55
  task_file: TaskGraph2

ZLQ:
  robot_num: 0
  velocity: 0 # m/h
  move_power: 0.0
  move_temperature: 0.0
  cold_down_temperature: -30 # origin -10.0
  cold_down_power: 0.1
  max_temperature: 80
  task_file: ZLQTask